frontend/src/components/ApplicationForm/ApplicationFormComments.tsx

```diff
@@ -1,46 +1,65 @@
-import * as React from 'react';
-
 import { Checkbox, SkjemaGruppe, TextareaControlled } from 'nav-frontend-skjema';
 import { Undertittel } from 'nav-frontend-typografi';
+import * as React from 'react';
+import {
+  _ADMIN_LABEL_NEEDS,
+  _CHECKBOX_NEEDS,
+  _LABEL_COMMENTS,
+  _LABEL_NEEDS,
+  _NEEDS_SUB_TITLE,
+} from './strings';
 
 interface IProps {
   updateApplicationFormData: (item: React.FormEvent) => any;
-  needs: boolean;
+  needs?: string;
+  hasNeeds: boolean;
+  comments?: string;
+  isAdmin: boolean;
 }
 
+const title = <Undertittel>{_NEEDS_SUB_TITLE}</Undertittel>;
+
 export const ApplicationFormComments: React.FunctionComponent<IProps> = (props) => {
-  const { updateApplicationFormData } = props;
+  const { updateApplicationFormData, hasNeeds, isAdmin } = props;
+  const comments = props.comments ? props.comments : '';
```

Comment:
Here you could use the more terse and readable: 
const comments = props.comments || '';

Reply:
fix'd